README.md

```
@@ -1,258 +1,62 @@
-  watch
-  [Implement Inversion of Control](https://egghead.io/lessons/egghead-implement-inversion-of-control?pl=kent-s-blog-posts-as-screencasts-eefa540c&af=5236ad)
-- The more experience you have with building React abstractions, the more
-  helpful this workshop will be for you.
+Acquaint yourself with my blog post on [Inversion of Control](https://kentcdodds.com/blog/inversion-of-control), or watch the video [Implement Inversion of Control](https://egghead.io/lessons/egghead-implement-inversion-of-control?pl=kent-s-blog-posts-as-screencasts-eefa540c&af=5236ad). The more experience you have with building React abstractions, the more beneficial this workshop will be.
 
 ## System Requirements
 
-- [git][git] v2.13 or greater
+The following need to be in your `PATH`:
+
+- [git][git] v2.13 or higher
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "git"

(MD052, reference-links-images)